src/vs/workbench/contrib/editSessions/browser/editSessions.contribution.ts

```diff
@@ -537,8 +543,9 @@ export class EditSessionsContribution extends Disposable implements IWorkbenchCo
 
 		if (getVirtualWorkspaceLocation(this.contextService.getWorkspace()) !== undefined && isNative) {
 			items.push(new ContinueEditSessionItem(
-				localize('continueEditSessionItem.openInLocalFolder', 'Open In Local Folder'),
+				'$(folder) ' + localize('continueEditSessionItem..v2', 'Open in Local Folder'),
```

Comment:
Not sure if you meant `openInLocalFolder.v2`

Reply:
Good catch, thanks!